docs/source/community/host-guide.md

```diff
@@ -0,0 +1,121 @@
+
+(If using this document as a template, replace this section with a list of topics that are discussed at your meeting.)
+
+## &#x1F4AA; Host Responsibilities
+
+First and foremost, the host should facilitate the flow of a meeting. That means:
+
+- [ ] Going through the agenda:
+  - *Do time checks/pause + move on to hit all items when there are time concerns*
+- [ ] Guiding discussion:
+  - *Read chats aloud for participants + the recording*
+- [ ] Maintaining order / Making sure everyone has a chance to speak:
+  - *Interject when someone has their hand raised*
+- [ ] Mention/link to the Jupyter Code of Conduct and how everyone is bound to it (including the host)
+
+Partner with another contributor (a meeting facilitator) to do the following:
+
+- [ ] Log in as the “Project Jupyter” host account for meeting management perks (removing spammers, muting and recording, etc.), [request access](mailto:security@ipython.org) from the Jupyter Security Team.
+- [ ] Record the meeting, if that is intended. The host starts and stops the recording. These are posted to YouTube.
+- [ ] Remove unwelcome recording bots:
+  - Per the Jupyter community page, participants
+  should not record meetings on their own. The host should ask participants
+  that look like AI recording bots:
+  - To identify themselves and state whether they're recording
+  - If they get no response from the account, the host should assume
+    it's recording, and remove it from the meeting.
+
+## &#x270f; Suggested Meeting Script and Checklist 
+
+### &#x23F3; Before the Meeting Starts
+
+- [ ] Log in to Zoom from the Project Jupyter account (request access by emailing [security@ipython.org](mailto:security@ipython.org)). This is necessary in order to record and manage meeting participants.  Alternatively, provided that someone who has host privileges logs onto the call they can grant host privileges in real time at the start of the meeting. 
+*from the Jupyter Security group if needed*)
+- [ ] Make sure you have a stable internet
+connection, a good quality microphone, and a quiet environment.
+- [ ] Check the [meeting agenda](insert agenda)
+  - *The agenda is a collaboratively created document made by the participants.*
+    *Ensure a section for today's meeting is created, with today's date and a table*
+    *for participants to record their names (which can be copied/pasted from lower*
+    *entries). The agenda will also be used as meeting minutes.*
+
+&#x1F517; *Links to copy/paste into the meeting chat:*
+
+- Weekly Meeting Notes: [Insert your team’s notes here]
+- Jupyter Code of Conduct: https://jupyter.org/governance/conduct/code_of_conduct.html
+- [Subproject Name]Team Compass: 
+
+### &#x1F399; Start the Meeting! 
+
+> Hello and welcome to our [day, month, year] Jupyter [TOPIC_HERE] call. I’m
+> [host name] and I’ll be your host today.
+> This is a place to for contributors to connect with each other
```

Comment:
Should we clarify what 'contributors' mean here? It sounds like it's for code contributors and maintainers only, which sounds exclusionary. The meetings are open to public, so we should encourage anyone to bring up issues, questions and suggestions